sc-poc/managers/configman/adapter/file/operations.go

```diff
@@ -24,7 +25,15 @@ func (f *File) List(gvr schema.GroupVersionResource) (*unstructured.Unstructured
 		return list, nil
 	}
 
-	list = common.ConvertToList(sources)
+	filteredSources := make([]*unstructured.Unstructured, 0)
+	for _, src := range sources {
+		labels := src.GetLabels()
+		if labels["space-cloud.io/package"] == listOptions.Labels["space-cloud.io/package"] {
```

Comment:
Don't just match one label. Match all labels provided to you in the `labels` map